script/Arcueid/Day 2/02_00_ARC02_4.txt

```diff
@@ -11,12 +11,12 @@
 [sha:543352fc739fcbda0fcd57f1fea80ba0d554d894]{
 -- Page 10013, Offset 3067.
 -- そのせいだろうか。どこに向かう予定だったのか見失っている気がする。
-I wonder if it's because of that. I feel like I've lost track of where I was going.
+I wonder if that's the reason... I feel like I've lost track of where I was going.
 }
 [sha:6f8c930ddc61fc2c5d3d20f135f302294ee608ab]{
 -- Page 181, Offset 3068.
 -- 夜は漏れ出した水のように街を<覆|おお>っている。
-The night is covering the city like water that has leaked out.
+The night oozes over the city like spilled water.
```

Comment:
as much as I like the imagery of "the night oozes over the city", "ooze" is a word I'd say doesn't fit well with water similes because of its viscous nature. 
I think "the night covers the city like spilled water" is cleaner for this reason.